src/manage_sql.c

```diff
@@ -10176,6 +10176,66 @@ alert_script_exec (const char *alert_id, const char *command_args,
   return 0;
 }
 
+/**
+ * @brief Write data to a file for use by an alert script.
+ *
+ * @param[in]  directory      Base directory to create the file in
+ * @param[in]  filename       Filename without directory
+ * @param[in]  content        The file content
+ * @param[in]  content_size   Size of the file content
+ * @param[in]  description    Short file description for error messages
+ * @param[out] file_path      Return location of combined file path
+ *
+ * @return 0 success, -1 error
+ */
+static int
+alert_write_data_file (const char *directory, const char *filename,
+                       const char *content, gsize content_size,
+                       const char *description, gchar **file_path)
+{
+  gchar *path;
+  GError *error;
+
+  if (file_path)
+    *file_path = NULL;
+
+  /* Setup certificate file */
```

Comment:
I guess this comment is for an old version because this function is not specifically about certs.

Reply:
Yes, this is an oversight from copying the code which I'll fix.